### DISCUSSION TITLE
Integrating  library to an apktool project

### DISCUSSION THREAD

**Discussion by acropolis914** [iBotPeaches/Apktool#3639] [Q&A] [answered]
I want to decompile an old apk from gingerbread era and recompile it with material ui. 

How do I integrate the android material library to an apktool decoded apk. Is that even possible?

If so, are there any recommended readings/docs on how to do it?

**Comment by iBotPeaches** [comment, marked as answer]
That I'm afraid sounds like a solid amount of work. I'm not aware of a guide for that.

**Reply by acropolis914** [reply]
I see thanks!
